src/components/Dialog.jsx

```diff
@@ -2,19 +2,19 @@
 import React  from 'react';
 import '../../sass/components/_ciq-dialog.scss';
 
-const chartControlHeight = 47;
 const Dialog = ({
     open,
     children,
     onContainerClick,
     className,
     isMobile,
     isFullscreen,
+    chartContainerHeight,
+    chartHeight,
 }) => {
     const inner_style = (isMobile && open) ? {
         // reduce chart control panel height except in full screen mode
-        height: `${window.innerHeight - (isFullscreen ? 0 : chartControlHeight)}px`,
-        width: `${window.innerWidth}px`,
```

Comment:
@shayanbinary do you have a reason for adding this? It seemed to work fine without it.